### PR TITLE
[TASK] Write information on Page module for Developers and Integrators

### PR DESCRIPTION
**Pull request by linawolf**
How to use the Page module for editing is described in the editors tutorial.

Releases: main, 13.4, 12.4

### CODE REVIEW

File: Documentation/Concepts/Backend/PageModule/Index.rst [hunk 1]
```diff
@@ -1,72 +1,99 @@
...
+:ref:`plugins <page-plugins>` are ordered into groups.
 
-   Page content as seen in the frontend
+You can use the page TSconfig setting
+:ref:`mod.wizard.newContentElement.wizardItems <t3tsconfig:pagenewcontentelementwizard>`
+to hide or edit content elements displayed here. For example you can hide the
+"HTML" content element supplied by :composer:`typo3/cms-fluid-styled-content`:
 
+..  code-block:: typoscript
+    :ref:`EXT:site_package/Configuration/page.tsconfig`
```
Comment (Alagts) [inline Documentation/Concepts/Backend/PageModule/Index.rst:52]:
```suggestion
..  code-block:: tsconfig
    :caption: EXT:site_package/Configuration/page.tsconfig
```

File: Documentation/Concepts/Backend/PageModule/Index.rst [hunk 2]
```diff
@@ -1,72 +1,99 @@
...
 
-.. todo: Mention backend_layouts here?
+New content elements can also be created without relying on third party
+extensions. You need basic knowledge on :ref:`tca`, :ref:`typoscript`, and
+:ref:`templating with Fluid <fluid-templates>`. You have to use PHP for some
+basic configurations but need no in-depth knowledge of programming.
 
-The Page module shows a preview of each content element, together
-with some action icons. Here is a quick overview:
+See :ref:`Create a custom content element type <t3coreapi:adding-your-own-content-elements>`
```
Comment (Alagts) [inline Documentation/Concepts/Backend/PageModule/Index.rst:73]:
```suggestion
See :ref:`create a custom content element type <t3coreapi:adding-your-own-content-elements>`.
```

File: Documentation/Concepts/Backend/PageModule/Index.rst [hunk 3]
```diff
@@ -1,72 +1,99 @@
...
+:composer:`wsr/myleaflet`, event management with registration:
+:composer:`derhansen/sf-event-mgt-contentelements` and many more.
 
+Chapter :ref:`how-to-find-extensions` covers searching for suitable extensions.
 
-Modifying existing content or adding new content is covered in the
-:ref:`Editors Tutorial <t3editors:content-elements>`.
+Usually a PHP class called a "controller" manages
+the functionality and display of the plugin. To create a custom plugin you
+need some experience in PHP programming, dealing with databases etc.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/PageModule/Index.rst:92]:
```suggestion
need some experience in PHP programming and dealing with databases etc.
```

File: Documentation/Concepts/Backend/PageModule/Index.rst [hunk 4]
```diff
@@ -1,72 +1,99 @@
...
-#. **Insert** a new content element below the current one
+A plugin is a special kind of content element. It typically provides dynamic or
+interactive functionality. Many third party extensions offer ready to use
+plugins for a wide range of functionality. For example plugins to display news:
+:composer:`georgringer/news`, plugins to perform searches:
+:composer:`apache-solr-for-typo3/solr`, to display Open Street maps:
+:composer:`wsr/myleaflet`, event management with registration:
+:composer:`derhansen/sf-event-mgt-contentelements` and many more.
 
+Chapter :ref:`how-to-find-extensions` covers searching for suitable extensions.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/PageModule/Index.rst:88]:
```suggestion
The chapter :ref:`how-to-find-extensions` covers searching for suitable extensions.
```

File: Documentation/Concepts/Backend/PageModule/Index.rst [hunk 5]
```diff
@@ -1,72 +1,99 @@
...
-============
-Pages module
-============
+==============================
+"Page" backend module in TYPO3
+==============================
 
-Site content can be structured in many ways.
+The :guilabel:`Web > Page` module is used by the editors of the site to add
+and modify content elements on the page.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/PageModule/Index.rst:10]:
```suggestion
and modify content elements on the page. A content element 
in TYPO3 is a bundle of pre defined TCA (Table Configuration Array) 
backend fields that we can 
add into our HTML code that is displayed on our website. A 
TCA backend field can be added to a content element. Every time 
a TCA field is filled out and saved, this value is 
stored in the database against this field and content element. And 
every time you add this content element in the backend you can 
fill out the same bundle of fields in the backend. The goal is to 
display the whole website with content elements, except the header 
and footer. 
```

Comment (linawolf) [reply Documentation/Concepts/Backend/PageModule/Index.rst:10]: Can we move that text of yours to its own concept page or into the glossary? It is out of scope here.

**Pull request merged**